### PR TITLE
[disaster] some more optimizations

### PR DESCRIPTION
**Pull request by chejennifer**
- remove child place fetch
- remove unnecessary blocking calculation when adding map points
- remove unnecessary node geojson call

### CODE REVIEW

File: server/routes/disasters.py [hunk 1]
```diff
@@ -66,6 +67,12 @@ def disaster_dashboard(place_dcid=None):
     dashboard_config.metadata.contained_place_types.update(
         place_metadata.contained_place_types_override)
 
+  child_places = place_api.child_fetch(place_dcid)
```
Comment (beets) [inline server/routes/disasters.py:70]: we could move this to a helper so it's returned in lib_subject_page_config.place_metadata (or I can do it, if this goes in first).

File: static/js/chart/draw_d3_map.ts [hunk 2]
```diff
@@ -516,20 +516,23 @@ export function addMapPoints(
...
-      const pathClientRect = (
-        paths[idx] as SVGPathElement
-      ).getBoundingClientRect();
-      minRegionDiagonal = Math.sqrt(
-        Math.pow(pathClientRect.height, 2) + Math.pow(pathClientRect.width, 2)
-      );
-    });
-  const minDotSize = minDotRadius || Math.max(minRegionDiagonal * 0.02, 1.1);
+  let minDotSize = minDotRadius;
+  if (!minDotSize) {
```
Comment (beets) [inline static/js/chart/draw_d3_map.ts:520]: could you add a comment for why this is important here so we don't undo this in the future

Comment (chejennifer) [reply static/js/chart/draw_d3_map.ts:522]: done

File: server/routes/disasters.py (hunk 1 above)
Comment (chejennifer) [reply server/routes/disasters.py:71]: since I think this will be going in first, I added a TODO for you to move this to a helper

**Pull request merged**